zosconnectee/index.js

```diff
@@ -1,3 +1,8 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
```

Comment:
@dhmlau  why not 2019? This goes for all instances of 2018.

Reply:
AFAIK, the 2nd year in this copyright statement is the year where it's last updated.

Reply:
I see, ok. :)